frida_tools/apk.py

```diff
@@ -191,18 +191,26 @@ def insert_debuggable(self, name: int, resource_map: ResourceMap) -> None:
 
         # Some parts of Android expect this to be sorted by resource ID.
         attr_offset = None
+        replace = False
         for insert_pos in range(self.attribute_count + 1):
-            attr_offset = 0x24 + 20 * insert_pos
+            attr_offset = 0x24 + struct.calcsize(self.ATTRIBUTE_FORMAT) * insert_pos
```

Comment:
```suggestion
            attr_offset = 0x24 + insert_pos * struct.calcsize(self.ATTRIBUTE_FORMAT)
```

Nit: Existing code has the quantity first, size second.